.github/workflows/tests.yml

```diff
@@ -220,6 +230,16 @@ jobs:
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
           skip_existing: true
+      - name: Set git config
+        run: |
+          git config --global user.email "github@action.com"
```

Comment:
these are weird :) I assume you had to put them there, but maybe explain the reason in a comment?